porcupine-core/src/System/TaskPipeline/PTask.hs

```diff
@@ -109,14 +110,17 @@ logTask = unsafeLiftToPTask $ \(sev, s) -> logFM sev $ logStr s
 ptaskRequirements :: Lens' (PTask m a b) (LocationTree VirtualFileNode)
 ptaskRequirements = splittedPTask . _1
 
+ptaskRunnable :: Lens (PTask m a b) (PTask m a' b')
+                      (RunnablePTask m a b) (RunnablePTask m a' b')
+ptaskRunnable = splittedPTask . _2
```

Comment:
For the records: "splitted" is weird. Prefer "split".

Reply:
A lot of lenses end with `-ed`: `traversed`, `mapped` etc

Reply:
Yes, but the past participle of "split" is "split". While "splitted" technically exist, it's an archaic word which nobody uses. It was just a general remark about English, nothing more. Nothing important either.